core/embed/rust/src/ui/layout_caesar/component/frame.rs

```diff
@@ -35,6 +37,11 @@ where
         self
     }
 
+    pub fn with_menu(mut self, has_menu: bool) -> Self {
```

Comment:
nit: IIUC the actual menu is not owned by the `Frame`, but by its child (e.g. `ShowMore::<Paragraphs<ParagraphVecShort>>` in TS3), right?

Maybe we should move the "menu-related" infomation in LayoutContent from top-level to its owner, and use `UnstructuredJSONReader.find_unique_value_by_key` to look it up?